docs/specs/validations/metadataContentValidations.yml

```diff
@@ -2070,24 +2072,21 @@ inputs:
   b.md: |
     hello
     ![alt text](pig2.jpg)
-  c.md: |
-    hello
-    [!include[](b.md)]
-  d.md: |
-    hello
-    [!include[](b.md)]
+    <img src="pig3.jpg" alt="alt text"></img>
+    :::image type="content" source="pig4.jpg" alt-text="alt text":::
 outputs:
   a.json:
-  c.json:
-  d.json:
   pig1.jpg:
   pig2.jpg:
+  pig3.jpg:
+  pig4.jpg:
   .errors.log: |
     {"message_severity":"suggestion","log_item_type":"user","code":"image-alt-text-duplicated","message":"Alt text 'alt text' is duplicated. Within an article, alt text must be unique.","file":"a.md","line":2,"end_line":2,"column":1,"end_column":1}
-    {"message_severity":"suggestion","log_item_type":"user","code":"image-alt-text-duplicated","message":"Alt text 'alt text' is duplicated. Within an article, alt text must be unique.","file":"b.md","line":2,"end_line":2,"column":1,"end_column":1}
-    
+    {"message_severity":"suggestion","log_item_type":"user","code":"image-alt-text-duplicated","message":"Alt text 'alt text' is duplicated. Within an article, alt text must be unique. Duplicated altText is in included files: 'a.md(4,1) -> b.md(2,1)'.","file":"b.md","line":2,"end_line":2,"column":1,"end_column":1}
```

Comment:
how about report the parent info at main error message just like what heading validation does

Reply:
OK. Got it.